### PR TITLE
chore(rln-relay): Use kilic rln in Docker

### PR DESCRIPTION
**Pull request by rymnc**
Preserve existing changes, but use kilic's rln instead of zerokit

### CODE REVIEW

File: shell.nix [hunk 1]
```diff
@@ -12,7 +12,7 @@ pkgs.mkShell {
     which # 2.21
     rustc # 1.63.0
   ] ++ lib.optionals stdenv.isDarwin [ 
-    libiconv
+    libiconv
```
Comment (jakubgs) [inline shell.nix:15]: Unnecessary space.

**Review by jakubgs — APPROVED** [review]

File: shell.nix (hunk 1 above)
Comment (rymnc) [reply shell.nix]: Addressed in [4512088](https://github.com/status-im/nwaku/pull/1240/commits/45120888440c9264b1952fa8ecb081be47c1b548)

File: Dockerfile [hunk 2]
```diff
@@ -29,10 +20,9 @@ RUN make -j$(nproc) deps RLN="$RLN"
 
 # Build the final node binary
 RUN make -j$(nproc) $MAKE_TARGET NIMFLAGS="$NIMFLAGS" RLN="$RLN"
```
[the remark this answers is not in the document]
Comment (rymnc) [reply Dockerfile]: Addressed in [25d3d1c](https://github.com/status-im/nwaku/pull/1240/commits/25d3d1c05fe379757a85b10f3ce058636980cf0e)

**Comment by rymnc** [comment]
Docker build fails on this PR

**Pull request closed without merging**